wordpress-mobile/gutenberg-mobile
# isSelected handling on PlainText-backed blocks

Fix #113 

This updates the GB hash to use the `isSelected` prop as an indication for the PlainText-backed blocks (currently the `More` and `Code` blocks) to gain focus when such a prop is `true`.

Related Gutenberg PR: https://github.com/WordPress/gutenberg/pull/11989

Before, the focus would not be gotten when a new "PlainText"-backed block was inserted:
![cursor_after_insert_okno](https://user-images.githubusercontent.com/6597771/48631935-6e557900-e99e-11e8-8fe3-336308eb2be2.gif)

Now, the cursor appears blinking after insertion:
![cursor_after_insert_ok](https://user-images.githubusercontent.com/6597771/48631977-83320c80-e99e-11e8-982d-ae6040680836.gif)

To test:
1. make sure to select a block (tap on a block and see the up/down arrows appear, etc)
2. now tap on the `+` inserter icon
3. choose one of Code or More blocks (the only ones that currently are backed by `PlainText `implementation)
4. observe the new block is inserted, the block is shown as selected (it has the up/down arrows and such) and the cursor starts blinking there (that is, it got the focus).
